client/post-editor/post-editor.jsx

```diff
@@ -252,7 +252,6 @@ export const PostEditor = React.createClass( {
 								onDismissClick={ this.hideNotice } />
 							<EditorActionBar
 								isNew={ this.state.isNew }
-								onPrivatePublish={ this.onPublish }
```

Comment:
No these are usefull, I guess this one has been moved to `EditorVisibility` and `Sidebar and we just forgot to clean if from here.